add extra information into exception reporting (#373)
riscv/riscv-cheri: src/hypervisor-integration.adoc

.Virtual supervisor trap default capability register
include::img/vstdcreg.edn[]

[#vstval,reftext="vstval"]

> ==== Virtual Supervisor Trap Value Register (vstval)

Reviewer: ```suggestion
=== Virtual Supervisor Trap Value Register (vstval)
```

> ==== Virtual Supervisor Trap Value Register (vstval)

Reviewer: Do we need this section at all? The H spec clearly states that these registers are analogous to the S-mode versions, and nothing is changing between stval vs vstval, so I did not add them initially

Author: well - if we have one we should probably have all of them.
In practice it's enough to say that stval is updated and vstval is updated accordingly which would shorten the doc

The <<vstval>> register is an SXLEN-bit read-write register.

When a fault is taken into VS-mode <<vstval>> is updated as for <<stval>> for all CHERI memory access traps.

.Virtual supervisor trap value register
[#vstval-format]
include::img/vstvalreg.edn[]

[#vstval2,reftext="vstval2"]

> ==== Virtual Supervisor Trap Value Register 2 (vstval2)

Reviewer: ```suggestion
=== Virtual Supervisor Trap Value Register 2 (vstval2)
```

> The <<vstval2>> register is an SXLEN-bit read-write register.

Reviewer: ```suggestion
The <<vstval2>> register is a VSXLEN-bit read-write register.
```
